prep_data.R

```diff
@@ -125,19 +124,24 @@ population$country <- sub("Syrian Arab Republic", "Syria", population$country)
 population$country <- sub("Venezuela\\, RB", "Venezuela", population$country)
 population$country <- sub("Yemen\\, Rep\\.", "Yemen", population$country)
 

```

Comment:
This can be part of another PR, but will we add a boolean of whether it is an island country to this dataset so we can easily filter on it? Or will we do that later? If you add a TODO I can make a PR to add that in

Reply:
For sure, let me add a todo.